cluster/userdata-master.yaml

```diff
@@ -166,11 +166,11 @@ write_files:
 
       KUBECTL="docker run --net host -v /srv/kubernetes/manifests:/manifests:ro quay.io/coreos/hyperkube:v1.4.4_coreos.0 /hyperkube kubectl"
 
-      for manifest in kube-dns-de.yaml kube-dashboard-de.yaml heapster-de.yaml secretary-de.yaml mate-de.yaml; do
+      for manifest in kube-dns-de.yaml kube-dashboard-de.yaml heapster-de.yaml secretary-de.yaml mate-de.yaml prometheus-node-exporter-de.yaml; do
         ${KUBECTL} apply -f /manifests/$manifest
       done
 
-      for manifest in kube-dns-svc.yaml heapster-svc.yaml kube-dashboard-svc.yaml; do
+      for manifest in kube-dns-svc.yaml heapster-svc.yaml kube-dashboard-svc.yaml prometheus-node-exporter-svc.yaml k8s-api-ro-nginx-config-cm.yaml; do
```

Comment:
probably makes sense to use `*-svc.yaml` now instead of listing everything :smirk: